ckan/lib/helpers.py

```diff
@@ -2834,9 +2872,21 @@ def load_plugin_helpers():
 
     helper_functions.clear()
     helper_functions.update(_builtin_functions)
+    chained_helpers = defaultdict(list)
 
     for plugin in reversed(list(p.PluginImplementations(p.ITemplateHelpers))):
```

Comment:
we don't reverse the plugins for chained helpers because we want "first plugin wins" like for templates, actions and other plugins.